desktop/cmp/input/Select.js

```diff
@@ -664,7 +672,8 @@ const cmp = hoistCmp.factory(
         if (model.asyncMode) {
             rsProps.loadOptions = model.doQueryAsync;
             rsProps.loadingMessage = model.loadingMessageFn;
-            if (model.renderValue) rsProps.defaultOptions = [model.renderValue];
+            if (model.queryForDefaultOptions) rsProps.defaultOptions = true;
+            if (model.renderValue && !model.queryForDefaultOptions) rsProps.defaultOptions = [model.renderValue];
```

Comment:
Not 100% sure. Seems like the use of this prop would indicate that the options should be populated like this regardless of existing input value, but a little weird in the sense that it's possible this existing value is NOT included in the returned options.